content/en/os/1.13.x/login/regaining-access/index.markdown

```diff
@@ -0,0 +1,90 @@
++++
+title = "Regaining Access to a Bottlerocket Node After Disabling the Control and Admin Containers"
+type = "docs"
++++
+
+## Introduction
+
+The standard way to access a shell on a Bottlerocket node is to use either the admin container or the control container.
+In some cases, both the admin and control containers may be disabled.
+There is a way to regain access to a Bottlerocket node in such a scenario.
```

Comment:
make this sentence a clause of the proceeding sentence.

Reply:
Is a semicolon between the two acceptable, or does that not solve the problem?

Alternatively: "There is a way to regain access to a Bottlerocket node when both the admin and control containers are disabled."
Thoughts?

Reply:
Maybe invert it: "In cases were both the admin and control ... you can still regain access to the Bottlerocket node" (put in your own words though)

Reply:
Oh! That's much clearer.